pydicom/waveforms/numpy_handler.py

```diff
@@ -135,7 +135,7 @@ def generate_multiplex(
                 sensitivity = ch.get("ChannelSensitivity", 1.0)
                 correction = ch.get("ChannelSensitivityCorrectionFactor", 1.0)
                 arr[..., jj] = (
-                    (arr[..., jj] + baseline) * sensitivity * correction
+                    arr[..., jj] * sensitivity * correction  + baseline
                 )
```

Comment:
Looks correct to me, but I would to have someone else have a look with a better understanding of waveforms (@scaramallion ?). I guess it is a bit diffcult to write a test here...

Reply:
I did try to verify the math (not something I've had experience with).  The standard does note that [baseline is in sensitivity units](https://dicom.nema.org/medical/Dicom/2016b/output/chtml/part03/sect_C.10.9.html#table_C.10-10) (verifying the comment in #1667: "It is already in "sensitivity" units").  It is also a VR of `DS` in the dictionary, which makes sense, vs an int matching the raw data type.

Note that the [pydicom docs on this](https://pydicom.github.io/pydicom/stable/tutorials/waveforms.html#decoding-waveform-data) also include this exact equation in the text, so also need to be updated.